vitest.shared.mts

```diff
@@ -0,0 +1,21 @@
+import * as path from 'path';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import tsconfigPaths from 'vite-tsconfig-paths';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { defineConfig, configDefaults } from 'vitest/config';
+
+export default defineConfig({
+  plugins: [
+    tsconfigPaths({
+      projects: [path.resolve(__dirname, 'tsconfig.json')],
+    }),
+  ],
+  test: {
+    exclude: [...configDefaults.exclude],
+  },
+  resolve: {
+    alias: {
+      '@chainflip/processor': path.resolve(__dirname, 'node_modules/@chainflip/processor/dist'),
```

Comment:
do you remember why this was needed? i think `@chainflip/processor` should be a normal package

Reply:
vitest refused to resolve the chainflip processor unless explicty se 🤷 .. Maybe something wrong with how we export?

Reply:
yeah thats possible - not a good sign 🙈